packages/cfd/src/Containers/cfd-compare-accounts/cfd-compare-accounts-button.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Button } from '@deriv/components';
+import { TModifiedTradingPlatformAvailableAccount } from 'Components/props.types';
+
+type TCFDCompareAccountsPlatformLabelProps = {
+    trading_platforms: TModifiedTradingPlatformAvailableAccount;
+};
+
+const CFDCompareAccountsButton = ({ trading_platforms }: TCFDCompareAccountsPlatformLabelProps) => {
+    return (
+        <div>
```

Comment:
There's no need for this empty div. You'd better use React.Fragment if necessary.